tests/backup/backup_locked_bucket_test.go

```diff
@@ -406,3 +406,233 @@ var _ = Describe("{LockedBucketResizeOnRestoredVolume}", func() {
+					periodicSchedulePolicyUid = uuid.New()
+					periodicSchedulePolicyUids = append(periodicSchedulePolicyUids, periodicSchedulePolicyUid)
+					periodicSchedulePolicyInfo := Inst().Backup.CreateIntervalSchedulePolicy(5, 15, 5)
+					err = Inst().Backup.BackupSchedulePolicy(periodicSchedulePolicyName, periodicSchedulePolicyUid, orgID, periodicSchedulePolicyInfo)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of periodic schedule policy of interval 15 minutes named [%s]", periodicSchedulePolicyName))
+					periodicSchedulePolicyUid, err = Inst().Backup.GetSchedulePolicyUid(orgID, ctx, periodicSchedulePolicyName)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching uid of periodic schedule policy named [%s]", periodicSchedulePolicyName))
+				})
+				Step("Create schedule backup after initializing volume resize", func() {
+					log.InfoD("Create schedule backup after initializing volume resize")
+					ctx, err := backup.GetAdminCtxFromSecret()
+					dash.VerifyFatal(err, nil, "Fetching px-central-admin ctx")
+					scheduleName = fmt.Sprintf("%s-schedule-%v", BackupNamePrefix, time.Now().Unix())
+					scheduleNames = append(scheduleNames, scheduleName)
+					labelSelectors := make(map[string]string)
+					err = CreateScheduleBackup(scheduleName, SourceClusterName, backupLocationName, backupLocationUID, []string{namespace},
+						labelSelectors, orgID, "", "", "", "", periodicSchedulePolicyName, periodicSchedulePolicyUid, ctx)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of schedule backup with schedule name [%s]", scheduleName))
+					firstScheduleBackupName, err := GetFirstScheduleBackupName(ctx, scheduleName, orgID)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the name of the first schedule backup [%s]", firstScheduleBackupName))
+				})
+				Step("Verifying backup success after initializing volume resize", func() {
+					log.InfoD("Verifying backup success after initializing volume resize")
+					ctx, err := backup.GetAdminCtxFromSecret()
+					dash.VerifyFatal(err, nil, "Fetching px-central-admin ctx")
+					firstScheduleBackupName, err := GetFirstScheduleBackupName(ctx, scheduleName, orgID)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching the name of the first schedule backup [%s]", firstScheduleBackupName))
+					_, err = backupSuccessCheck(firstScheduleBackupName, orgID, 0, 0, ctx)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying the success of recent backup named [%s]", firstScheduleBackupName))
+					allScheduleBackupNames, err := Inst().Backup.GetAllScheduleBackupNames(ctx, scheduleName, orgID)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching all schedule backups %v", allScheduleBackupNames))
+					log.InfoD("Waiting for 15 minutes for the next schedule backup to be triggered")
+					time.Sleep(15 * time.Minute)
+					recentBackupName, err := GetLatestScheduleBackupName(ctx, scheduleName, orgID)
```

Comment:
The function GetLatestScheduleBackupName may not always indicate the next schedule backup. It is recommended to use GetOrdinalScheduleBackup instead and set the ordinal to currentScheduleBackupCount + 1. For further details, please check PR #1191.

Reply:
If you are planning to use GetOrdinalScheduleBackup, you might want to change the variable name accordingly.

Reply:
Thanks!